cmd/kube-scheduler/app/server.go

```diff
@@ -284,7 +294,7 @@ func newHealthzHandler(config *kubeschedulerconfig.KubeSchedulerConfiguration, s
 }
 
 // NewSchedulerConfig creates the scheduler configuration. This is exposed for use by tests.
-func NewSchedulerConfig(s schedulerserverconfig.CompletedConfig) (*scheduler.Config, error) {
+func NewSchedulerConfig(s schedulerserverconfig.CompletedConfig) (*factory.Config, error) {
```

Comment:
I checked and this is still in use by `test/integration`. We'll have to clean it up (delete it) later.

Reply:
need I clean it up in this pr? or next?

Reply:
Not in this PR